SpotlightDownloader/Program.cs

```diff
@@ -17,239 +17,148 @@ class Program
 
         static void Main(string[] args)
         {
-            //Configure System.Net to use TLS 1.2 - disabled by default for .NET 4.0 - Stackoverflow 33761919 - Issue #22
-            System.Net.ServicePointManager.SecurityProtocol = (System.Net.SecurityProtocolType)3072; //Tls12
+            ParsedArguments parsed = null;
+            try
+            {
+                parsed = ArgumentParser.Parse(args);
+            }
+            catch (ArgumentException ex)
+            {
+                if (ex.Message == "Show help")
+                {
+                    foreach (string str in new[]{
+                        " ==== " + Name + " v" + Version + " - By ORelio and its contributors - Microzoom.fr ====",
```

Comment:
You can remove the credit I left here and just put the repo URL instead.
```suggestion
                        " ==== " + Name + " v" + Version + " - https://github.com/ORelio/Spotlight-Downloader/ ====",
```